scripts/download/lima.mjs

```diff
@@ -7,7 +7,7 @@ import path from 'path';
 import { download, getResource } from '../lib/download.mjs';
 
 const limaRepo = 'https://github.com/rancher-sandbox/lima-and-qemu';
-const limaTag = 'v1.17';
+const limaTag = 'v1.19';
 
 const alpineLimaRepo = 'https://github.com/lima-vm/alpine-lima';
 const alpineLimaTag = 'v0.2.5';
```

Comment:
Hi, It's needed to update also the alpineLimaTag to v0.2.6 otherwise the old nerdctl version is used.

Reply:
@dschunack Thank you, you are correct! And the change to `limaTag` should be in #1423 and not here...

Reply:
Oh, it is already part of #1423, so just needs to be removed here

Reply:
Thank you, should be fixed now!